test/framework/options.py

```diff
@@ -2433,6 +2433,27 @@ def test_fixed_installdir_naming_scheme(self):
         app.gen_installdir()
         self.assertTrue(app.installdir.endswith('software/toy/0.0'))
 
+    def _assert_regexs(self, regexs, txt, assert_true=True):
+        for regex in regexs:
+            regex = re.compile(regex, re.M)
+            if assert_true:
+                self.assertTrue(regex.search(txt), "Pattern '%s' found in: %s" % (regex.pattern, txt))
+            else:
+                self.assertFalse(regex.search(txt), "Pattern '%s' NOT found in: %s" % (regex.pattern, txt))
+
+    def _run_mock_eb(self, args, stdout=True, stderr=False, do_build=False, raise_error=False, verbose=False, testing=False, strip=False):
```

Comment:
@migueldiascosta Please include a docstring when adding methods

Reply:
Also, why not always mock both `stdout` and `stderr`?

I would remove the `stdout` & `stderr` named arguments in this method, and always mock & return both?